### ISSUE TITLE
GPU widget, CPU widget enhancements

### ISSUE THREAD

**Issue by StealthBravo** [tjmarkham/win10widgets#1]
Just like CPU, it'd be great to have a GPU widget to view current stats and load.

For CPU (and GPU, if added), I also think the addition of a temperature indicator would be a minor but very useful feature.

**Comment by NCopeman** [comment]
Adding onto the feature suggestions, I'd appreciate Disk R/W metrics in terms of MB/s, GPU 
Fan Speeds, and a per-core CPU utilisation view.

For reference, ![this](https://cloud.githubusercontent.com/assets/7776908/16830488/baf763d0-49e2-11e6-9f4e-7ca84b5fb407.png) is my current Rainmeter layout, using a modified version of Rolandazz. GPU/CPU stats are pulled using MSI Afterburner.

**Comment by danieltudosiu** [comment]
Another example would be [this](https://itunes.apple.com/us/app/monity/id915542151?mt=12) which if for Mac OSX.

What I think ppl want is basically the following stats, either numbers or graphs, per core/processor/gpu:
*Temperature
*Fan
*Load

For storage
*Multiple storage supported
*Disk Read/Write

For network
*Allow any interface to be selected and tracked for both read and write, and also maintain the load one

I liked your skin due to the fact that I searched for something to fit within Windows 10 Modern UI and replace my friends's monity.

You did a great job but there is a lot of space for improvement, KEEP IT GOING!

**Comment by mindleak** [comment]
Agreed with all of this. Attached is the kind of info I'd love to be able to display in your beautiful widgets.

![65cddb2785c7f5a18a024f36409fcbcc](https://cloud.githubusercontent.com/assets/16060889/17235701/c50aa64e-5508-11e6-81f7-6c58591cc5c4.jpg)

**Comment by Kamylow** [comment]
Hello there, adding a wish for that too. I use to use orbmu2k widgets with windows 7 and the week i upgraded i saw your work, it's very promising ! The one i miss the most is the Top Five one (http://orbmu2k.de/sidebar-gadgets/top-five-sidebar-gadget) as its name suggests it give a quick glance at the top 5 CPU and memory intensive apps. After is kind of looks like from Mindleak, just more minimalist. Hope you'll have time and/or the will to make something like that.
Nonetheless huge props to you sir.
![capture](https://cloud.githubusercontent.com/assets/20738607/17269723/a7d5ea2c-5650-11e6-9de0-c30a410a1f71.JPG)

**Comment by theSplund** [comment]
I'd really like to move on from Gadgets (sidebar.exe) but to do this then GPU, with temperature monitor/alarm (and multiple drives), is a must.

**Comment by seanvree** [comment]
I'd love this!

**Comment by nickakawoodstock** [comment]
This would be brilliant to add. Thanks in advance.

**Comment by fpp3** [comment]
I would love to have at least a temperature monitor so i can easily see all my gpu and cpu temps. Also (maybe this is a bit overkill, but, why not), there could be a separate combo, that implements all of the different temperatures and also shows current fan speed! Maybe im very obsessive with hw monitoring but, it would be cool, i think...
